packages/wear-os/README.md

```diff
@@ -110,8 +110,11 @@ export const demoTasks: Array<Task> = [
 
 > **Warning**: the data collection for a `WatchSensor` can only be started once, if `startDetectingFastWatchAccelerometerChanges` is executed after
 > `startDetectingWatchAccelerometerChanges` and while the collection is in progress, `startDetectingFastWatchAccelerometerChanges` will be ignored.
+> 
 > Therefore, if you want to dynamically change the collection's configuration while the collection is in progress, you will have to stop the collection
-> and start it again with the new desired configuration.
+> and start it again with the new desired configuration. However, due to the underlying communication APIs (i.e., Bluetooth), the order of the tasks **is not guaranteed**. 
+> This means that if the stop and start tasks are executed very close in time, the start task could be executed before the stop task in the smartwatch.
+> If you really need to change the collection's configuration, you should wait a certain amount of time (e.g., 1 second) after the execution of the stop task before executing the start task.
```

Comment:
How do you think they could implement this? Because the minimum time I can think they can delay a task is 1 minute, right? Or am I missing something?

Reply:
As we discussed, a way could be to use a task between the stop/start to add a delay. Other option could be to use a task to do the stop-wait-start using the client API. Right now this API is internal, so, if we need it in the future, I will export it then.

Reply:
Understood. Looks fair to me